src/messages.c.jinja

```diff
@@ -0,0 +1,47 @@
+// NOTE: This file has been generated automatically by “{{script}}”.
+//       Do not edit manually!
+
+#include <stdlib.h>
+#include "messages.h"
+
+static const struct xkb_message_entry xkb_messages[] = {
+    {% for entry in entries %}
+    { {#--#}{{ entry.message_code_constant}}{#, "{{entry.id}}", "{{entry.description}}"#}},
+    {% endfor %}
+    {0{#, NULL, NULL #}}
```

Comment:
Why is this needed BTW?

Reply:
The table is needed for the lookup. But in fact we do not use it in the main lib. I refactored it in the last version: I moved it to `tools/` and use the code only in the tool and the corresponding test.